interfaces/Portalicious/src/app/pages/projects-overview/components/project-metric-container/project-metric-container.component.html

```diff
@@ -1,10 +1,13 @@
+<dt class="px-2 pb-2 pt-1 txt-body-m">
```

Comment:
You're changing the project overview page with this change. I would advise against that - even if there is a refactor on the horizon. We shouldn't merge a regression, so I would rever the changes in this file.

Also, there is a component called `app-metric-tile` which I think contains the same thing you need here (I might be wrong, so double check), so maybe you wanna extrapolate that instead.

Whichever you end up using, if we end up using it in multiple pages, be sure to move it out of the `pages` folder, and into the common `src/app/components` folder.